Airways/RNAV/Upper/M195.txt

```diff
@@ -1,3 +1,2 @@
-LORKU LORKU MARUK MARUK
```

Comment:
Deleted line should still be in Upper definition, since LORKU-NARUK is FL245-FL460

Reply:
yh looks like i saved the files the wrong way around between upper and lower